packages/desktop/views/dashboard/Dashboard.svelte

```diff
@@ -141,6 +141,21 @@
 
 <Idle />
 <div class="dashboard-wrapper flex flex-col w-full h-full">
+    {#if $activeProfile?.network?.id !== NetworkId.Shimmer || $activeProfile?.network?.id !== NetworkId.ShimmerTestnet}
+        <div
+            class=" bg-blue-100 dark:bg-gray-800 flex flex-row gap-1 items-center justify-center py-2 font-400 text-13 leading-160 text-center"
+        >
+            <span class=" text-gray-800 dark:text-white max-w-lg"
+                >Please switch to the new IOTA Wallet for continued support and updates.
+            </span>
+            <a
+                href="https://chromewebstore.google.com/detail/iota-wallet/iidjkmdceolghepehaaddojmnjnkkija"
```

Comment:
does the link work? doesnt firefly prevent openening these external links? I think you have to add this URL to the list of allowed URLs